core/file_walker.go

```diff
@@ -40,17 +37,16 @@ func (f *File) UpdateSize() {
 
 type ReadDir func(dirname string) ([]os.FileInfo, error)
 
-func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[string]struct{}) *File {
-	var mutex sync.Mutex
+func WalkFolder(path string, readDir ReadDir, ignoredFolders map[string]struct{}) *File {
 	var wg sync.WaitGroup
-	c := make(chan bool, maxConcurrentScans)
-	root := getSubTreeConcurrently(path, parent, readDir, ignoredFolders, c, &mutex, &wg)
+	c := make(chan bool, runtime.NumCPU())
```

Comment:
It was faster for me this way than 64. I'm not sure if it is optimal though. I just thought that 64 is quite constant (not considering how much parallelism can given machine achieve)